### PR TITLE
chore: simplify PR template

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,49 +1,11 @@
 ---
 name: Pull Request Template
 about: Default pull request template
-title: ''
+title: '<type>[optional scope]: <description>'
```
Comment (VB-at-Bis) [inline .github/pull_request_template.md:4]: Shall we add list of types like it was before ( bug fix, new feature, breaking change, documentation only).
What is scope here? FE, BE, test, devops or something else?

Comment (matthieu-foucault) [reply .github/pull_request_template.md:4]: I can add some support documentation, following the example of Angular: https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type

**Review by VB-at-Bis — APPROVED** [review]
Looks good. Thanks for team agreement update!

**Comment by matthieu-foucault** [comment]
Looks like the PR template didn't work as expected... opening a new PR shortly

**Pull request merged**